### DISCUSSION TITLE
Where is the release progress and validation of each edk2 tag tracked

### DISCUSSION THREAD

**Discussion by sachinami** [tianocore/edk2#6111] [Q&A]
The release planning for edk2 tags is tracked here: https://github.com/tianocore/tianocore.github.io/wiki/EDK-II-Release-Planning

Similarly, is there a GitLab issue or Bugzilla ticket created for each release, to track activities such as:

- Request for patches to be merged after Hard Freeze
- Progress of release validation and analysis of any bugs observed
- Determining which items are to be listed in "New Features & Bug Fixes"
- Any delays in the schedule

**Comment by sachinami** [comment]
@lgao4 I would like to know if there is any specific process followed for the above.

**Reply by lgao4** [reply]
- The request will be sent to the mail list after hard freeze if the developer wants to merge his patch.
- The observed problem will be sent to the mail list if anyone finds it. 
- I will review the merged pull request to determine which one is feature or bug. I really suggest to add label for each PR to specify whether it is feature or bug. 
- The delay request is also sent to the mail list for discussion.
